server/container_create_linux_test.go

```diff
@@ -158,3 +164,96 @@ func TestAddOCIBindsCGroupRW(t *testing.T) {
 		t.Error("Cgroup mount not added with RO.")
 	}
 }
+
+func TestGetRequestedCDIDevices(t *testing.T) {
```

Comment:
can we use ginkgo instead of go test framework?

Reply:
I rewrote the tests using ginkgo/gomega and moved them over to `pkg/container/devices_test.go`.